PatternPal/PatternPal.LoggingServer/Protos/log.proto

```diff
@@ -0,0 +1,54 @@
+syntax = "proto3";
+
+option csharp_namespace = "PatternPal.LoggingServer";
+
+package log;
+
+// The greeting service definition.
+service Log {
+  // Sends a greeting
+  rpc Log (LogRequest) returns (LogReply);
```

Comment:
In the other protos I use the convention of postfixing a request with `Request`, and a response/reply with `Response`.

```suggestion
  rpc Log (LogRequest) returns (LogResponse);
```